src/connector/src/with_options.rs

```diff
@@ -204,6 +204,14 @@ pub trait WithPropertiesExt: Get + GetKeyIter + Sized {
 
     fn is_refreshable_connector(&self) -> bool {
         false
+        // TODO: enable this when implementation done
+        // self.get(UPSTREAM_SOURCE_KEY)
+        //     .map(|s| s.eq_ignore_ascii_case(BATCH_POSIX_FS_CONNECTOR))
+        //     .unwrap_or(false)
```

Comment:
Is it done?

Reply:
We enable this after the whole feature is done, i.e., in  #22695 (which has e2e test)